roles/neutron-common/templates/etc/neutron/plugins/ml2/ml2_plugin.ini

```diff
@@ -19,6 +19,9 @@ tenant_network_types = {{ neutron.tenant_network_type }}
 {% endif %}
 mechanism_drivers = {{ mechanism_drivers|join(',') }}
 
+{% set extension_drivers = ['port_security'] %}
+extension_drivers = {{ extension_drivers|join(',') }}
```

Comment:
```
neutron:
  extension_drivers:
    - port_security
```

The template already joins the entries with a ','. 

If a particular site wants to use a different set of extensions, they can override the variable and define the list however they want it.

Given that a site may wish to choose no drivers, we need to wrap the config entry around a check for the length of the variable, or test that an empty value for the config is acceptable (and then use a `default` filter to set the variable to an empty list`.

Make sure

`extension_drivers = `

is acceptable.